### PR TITLE
feat(deps)!: update prisma to v2.16.0

### PR DESCRIPTION
**Pull request by lvauvillier**
Fix snapshot and examples

### CODE REVIEW

**Comment by jorgenskogas** [comment]
@lvauvillier: from the lates prisma release
https://github.com/prisma/prisma/releases/tag/2.16.0

## The Prisma CLI moves from the `@prisma/cli` to the `prisma` npm package

Going forward, you can install the Prisma CLI via the `prisma` npm package:

```
npm install prisma --save-dev
```
or

```
yarn add prisma --dev
```
The reason for this is that a number of users were experiencing issues when running the `npx prisma` command. Without a local installation of the `@prisma/cli` package, this would invoke the Prisma 1 CLI leading to an error message. From now on, `npx prisma` is _always_ going to work.

We will also deprecate the `@prisma/cli` package. Please do a _find and replace_ across your codebase to transition over from `@prisma/cli` to `prisma`. To make this transition easier, we'll keep publishing updates to both packages for another month and plan to stop updating the `@prisma/cli` package with release `2.18.0`.

There are no changes to the `@prisma/client` npm package name.

File: examples/blog/package.json [hunk 1]
```diff
@@ -15,14 +15,14 @@
...
+    "@prisma/client": "2.16.0",
     "graphql": "^15.4.0",
     "graphql-yoga": "^1.18.2",
     "nexus": "^1.0.0",
-    "nexus-plugin-prisma": "^0.28.0"
+    "nexus-plugin-prisma": "^0.29.0"
   },
   "devDependencies": {
-    "@prisma/cli": "2.15.0",
+    "@prisma/cli": "2.16.0",
```
Comment (jasonkuhrt) [inline examples/blog/package.json]: This needs to become `prisma`.

File: examples/hello-world/package.json [hunk 2]
```diff
@@ -10,16 +10,16 @@
...
     "express": "^4.17.1",
     "graphql": "^15.4.0",
     "graphql-scalars": "^1.7.0",
     "nexus": "^1.0.0",
-    "nexus-plugin-prisma": "^0.28.0"
+    "nexus-plugin-prisma": "^0.29.0"
   },
   "devDependencies": {
-    "@prisma/cli": "2.15.0",
+    "@prisma/cli": "2.16.0",
```
Comment (jasonkuhrt) [inline examples/hello-world/package.json]: `prisma`

File: package.json [hunk 3]
```diff
@@ -46,12 +46,12 @@
   },
   "devDependencies": {
     "@prisma-labs/prettier-config": "0.1.0",
-    "@prisma/cli": "2.15.0",
-    "@prisma/client": "2.15.0",
-    "@prisma/fetch-engine": "2.15.0",
-    "@prisma/get-platform": "2.15.0",
-    "@prisma/migrate": "2.15.0",
-    "@prisma/sdk": "2.15.0",
+    "@prisma/cli": "2.16.0",
```
Comment (jasonkuhrt) [inline package.json]: `prisma`

**Review by jasonkuhrt — CHANGES_REQUESTED** [review]
Almost there! Thanks @lvauvillier

**Comment by lvauvillier** [comment]
PR updated. Thanks @jorgenskogas @jasonkuhrt

**Review by jasonkuhrt — APPROVED** [review]
🍊

**Comment by darrylyoung** [comment]
Thanks for stepping in and doing this, @lvauvillier. 🥇

**Comment by BjoernRave** [comment]
@lvauvillier yea thanks a lot for that. I was already worried I was stuck on `2.15` for a while.

**Pull request merged**